tsconfig.json

```diff
@@ -0,0 +1,73 @@
+{
+    "compilerOptions": {
+        /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+        /* Basic Options */
+        // "incremental": true,                   /* Enable incremental compilation */
+        "target": "esnext" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
+        "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */,
+        // "lib": [],                             /* Specify library files to be included in the compilation. */
+        "allowJs": true /* Allow javascript files to be compiled. */,
+        // "checkJs": true,                       /* Report errors in .js files. */
```

Comment:
Codacy found an issue: [Comment not allowed](https://app.codacy.com/gh/P2P-Develop/TopDomainChecker/pullRequest?prid=6364165)